strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/services/ArtifactManagementService.java

```diff
@@ -211,90 +208,36 @@ private long writeArtifact(RepositoryPath repositoryPath,
         {
             artifactEventListenerRegistry.dispatchArtifactUploadingEvent(repositoryPath);
         }
-        
+
         long totalAmountOfBytes = IOUtils.copy(is, os);
 
-        URI repositoryPathId = repositoryPath.toUri();
         Map<String, String> digestMap = aos.getDigestMap();
         if (Boolean.FALSE.equals(checksumAttribute) && !digestMap.isEmpty())
         {
             // Store artifact digests in cache if we have them.
-            addChecksumsToCacheManager(digestMap, repositoryPathId);
+            addChecksumsToCacheManager(digestMap, repositoryPath);
         }
 
         if (Boolean.TRUE.equals(checksumAttribute))
         {
-            byte[] checksumValue = ((ByteArrayOutputStream) aos.getCacheOutputStream()).toByteArray();
```

Comment:
we shouldn't drop this and all above logic related with checksum validation

Reply:
We will discuss it with @carlspring soon I hope

Reply:
Yeah, why has this been dropped? Has it been replaced by something else, or...? What's the reasoning?

Reply:
It has been replaced by 

`String checksumValue = ((ByteArrayOutputStream) aos.getCacheOutputStream()).toString(StandardCharsets.UTF_8.name());`